cargo/cargo_build_script.bzl

```diff
@@ -261,5 +287,5 @@ def cargo_build_script(
         crate_features = crate_features,
         version = version,
         build_script_env = build_script_env,
-        deps = deps,
+        data = data,
```

Comment:
You need to re-add `deps = deps` - deps still matter! :)

Reply:
Whoops, good catch - not sure how I managed to do that!